ReactNative/iTwinRNStarter/android/app/src/release/java/com/bentley/itwinrnstarter/ReactNativeFlipper.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
```

Comment:
Same license/copyright problem as the other flipper file.

Reply:
Put a Meta-specific MIT license file in the repository root named REACT_NATIVE_LICENSE, then update these comments to have that instead of LICENSE.